application/helpers/frontend_helper.php

```diff
@@ -1722,7 +1722,7 @@ function checkCompletedQuota($surveyid, $return = false)
             // Answers are the same in quota + an answer is submitted at this time (bPostedField)
             //  OR all questions is hidden (bAllHidden)
             $bAllHidden = QuestionAttribute::model()
-                ->countByAttributes(array('qid'=>$aQuotaQid), 'attribute=:attribute AND value=:value', array(':attribute'=>'hidden',':value'=>1)) == count($aQuotaQid);
```

Comment:
I confirm : this part didn't fix and was not related to 16909.

Reply:
Yes, this is a quota-triggered bug, I also submitted this branch

Reply:
https://bugs.limesurvey.org/view.php?id=16909

Reply:
Yes : but still:

1. Are  not related **Slow loading when the questionnaire has a large quota(500+)**
2. Change the way quota are done : this can broke existing survey
3. See [16925: Quota : enforced option (checked on all page)](https://bugs.limesurvey.org/view.php?id=16925)